### ISSUE TITLE
Insufficient focus outline on resources page

### ISSUE THREAD

**Issue by jchristgit** [python-discord/site#1400]
As pointed out in #1363:

While the selectable items in the filter section have an outline around the checkbox, this area is small and may be hard to notice.

The focus indicator should be larger, potentially outlining the entire item label.

![image](https://github.com/user-attachments/assets/69acb65c-592a-4f17-b096-f49e0a3b3d24)
